contracts/RelaySugar.vy

```diff
@@ -68,7 +75,19 @@ def __init__(_factory: address, _ve_sugar: address, _ve: address):
 
 @external
 @view
-def all() -> DynArray[Relay, MAX_COMPOUNDERS]:
+def all(_account: address) -> (DynArray[Relay, MAX_COMPOUNDERS], DynArray[RelayVeNFT, MAX_RESULTS]):
+  """
+  @notice Returns all AutoCompounders and account's deposits
+  @return Array of Relay structs, Array of account's deposits
+  """
+  autocompounders: DynArray[Relay, MAX_COMPOUNDERS] = self._autocompounders()
+  deposits: DynArray[Deposit, MAX_RESULTS] = self._deposits()
```

Comment:
@ethzoomer should we extend the `Relay` struct to include a list of account venfts (eg. `account_venft_ids: DynArray[uint, MAX_RESULTS]`)?

Feels a bit weird to have it separate and then have to manage the two datasets in the front-end.

Also, I'd suggest we not rely on the `VeSugar.byAccount()` or any other call there and instead just use direct `VotingEscrow` calls since it will require a lot less gas to fetch the account venfts.